keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2619,6 +2619,14 @@ Beta:
   - Automatically manages versions of mandatory/suggested configuration
   - Discrimates paginated LIST requests
 
+GA:
+
+- Satisfaction with LIST and WATCH support.
+- API annotations properly support strategic merge patch.
```

Comment:
What does this mean?

Reply:
This means we botched the field tags when we first wrote them.

Reply:
Oh, you literally mean the tags on fields? OK

Reply:
(I don't think this is a GA requirement personally, it's more a tactical need until we get SSA on in integration tests)